pset-functions.js

```diff
@@ -91,6 +118,26 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+function wordCount (phrase) {
+  const words = phrase.split(' ')
+  //  empty array for counting words
+  const wordCounter = {}
+
+  //  for each word in the phrase
+  words.forEach((word) => {
+    if (wordCounter[word] === undefined) {
+      //  debug
+      //  console.log ('word doesnt exist. adding word')
+      wordCounter[word] = 1
+    } else {
+      wordCounter[word] = wordCounter[word] + 1
+    }
+  })
+
+  return wordCounter
+}
+
+console.log(wordCount('Humpty Dumpty sat on a wall Humpty Dumpty had a great fall'))
 // Add your code above this line
```

Comment:
@sad3xoxo Great Job on this solution utilizing the string method `split()` to create an array with each word and loop through each element to check if the word exists and to increment the value if the word does exist.
Take a look at this approach [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L103)